src/commands/definition.ts

```diff
@@ -0,0 +1,39 @@
+import * as Discord from 'discord.js';
+
+const urban = require('relevant-urban');
+module.exports = {
```

Comment:
`module.exports = {}` is a [quite old (prior to ES6)](https://stackoverflow.com/questions/40294870/module-exports-vs-export-default-in-node-js-and-es6) and is a rather - not so good way of doing things, [this post](https://stackoverflow.com/questions/46677752/the-difference-between-requirex-and-import-x) also explains why. I highly recommend of changing this to:
```ts
const definition = {
  name: 'definition'
  // and so on
}

export { definition };
```

Well - technically you would want to restructure it a bit after having created the Command Interface anyways, which would change the above part (the export statement would stay the same)!